src/Fixer/Phpdoc/PhpdocVarWithoutNameFixer.php

```diff
@@ -20,6 +20,7 @@
 use PhpCsFixer\Preg;
 use PhpCsFixer\Tokenizer\Token;
 use PhpCsFixer\Tokenizer\Tokens;
+use const T_VAR;
```

Comment:
What is the reason for this import?

Reply:
PhpStorm settings I use in personal projects

`use` :allthethings:

Should've seen and removed that. Will do so now 😄